tests/integration_tests/modules/test_ssh_keysfile.py

```diff
@@ -77,13 +77,15 @@ def common_verify(client, expected_keys):
         # Ensure we haven't messed with any /home permissions
         # See LP: #1940233
         home_dir = '/home/{}'.format(user)
-        home_perms = '755'
+        # Home permissions aren't consistent between releases. On ubuntu
+        # this can change to 750 once focal is unsupported.
+        home_perms = '75'
```

Comment:
Can we do something a bit more explicit here?
 ?
```suggestion
        if ImageSpecification.from_os_image().release in ("xenial", "bionic"):
            home_perms = '755'
        else:
            home_perms = '750'
```